extensions/common/azure/azure-blob-core/src/main/java/org/eclipse/edc/azure/blob/api/BlobStoreApiImpl.java

```diff
@@ -150,7 +149,7 @@ public BlobAdapter getBlobAdapter(String accountName, String containerName, Stri
 
     private BlobAdapter getBlobAdapter(String accountName, String containerName, String blobName, BlobServiceClientBuilder builder) {
```

Comment:
interesting that these `getBlobAdapter` methods don't use the cache

Reply:
I see, but am unaware if "both" business logic makes sense together.